ingest/workflow/snakemake_rules/upload.smk

```diff
@@ -21,5 +21,5 @@ rule upload_to_s3:
         cloudfront_domain = config["upload"].get("s3", {}).get("cloudfront_domain", "")
     shell:
         """
-        ./bin/upload-to-s3 {input:q} {params.s3_dst:q}/{wildcards.remote_file_name:q} {params.cloudfront_domain}
+        ./vendored/upload-to-s3 {input:q} {params.s3_dst:q}/{wildcards.remote_file_name:q} {params.cloudfront_domain}
```

Comment:
Note the vendored `upload-to-s3` brings back the `--quiet` option that was removed in #31. 
I think we could just hard-code `--quiet` here since we don't expect to use Slack notifications.

Reply:
Done with 9956d1fbe0230a1442d4e4c65ddaaa0c9fe12de7.